include/edm4hep/GenToolInfo.h

```diff
@@ -0,0 +1,48 @@
+#ifndef EDM4HEP_GENTOOLINFO_H
```

Comment:
Should this go to the `utils`? Mainly because, e.g. the ParticleID helpers also live there.

Reply:
it could. The struct however I'd like to keep higher level.